### DISCUSSION TITLE
cd-rom?

### DISCUSSION THREAD

**Discussion by Zack13358** [keirf/flashfloppy#745] [Ideas]
Anybody know if there is something like this for cd/dvd-rom drives?

**Comment by ejona86** [comment]
That would be called an Optical Drive (or Disc) Emulator, abbreviated ODE. They have become reasonably common for the older CD-based consoles. They tend to be device-specific.

**Comment by aperezbios** [comment]
@Zack13358 without knowing which specific device/interface you're thinking about, these sorts of devices already exist on the SCSI side. Eight or nine years ago, SCSI2SD implemented support for CD-ROM emulation, and there are numerous other products, such as MacSD, BlueSCSI, and ZuluSCSI, which can emulate _SCSI_ CD-ROM drives, to varying extents.

**Reply by Zack13358** [reply]
@aperezbios I was talking about IDE/ATAPI, but thanks! I might be able to see if SCSI to ide adapters exist
